### DISCUSSION TITLE
Welcome to APsystems-EZ1-API-HomeAssistant Discussions!

### DISCUSSION THREAD

**Discussion by SonnenladenGmbH** [SonnenladenGmbH/APsystems-EZ1-API-HomeAssistant#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by elec2** [comment]
I just wanted to say a quick thank you. It's just amazing to see how a German youtuber can have such an influence on a Chinese company so that we can all use the device in local mode.

The integration was super easy to install and works flawlessly. Now I have quickly set the MAC in the router to the internet blacklist and everything is great. 

Thank you very much for this!

Next project for me: DIY energy buffer

**Reply by uTZFu** [reply]
Same here - thanks for your efforts and this great integration !!!!
